### ISSUE TITLE
Reference links crossing lines in blockquotes don't work.

### ISSUE THREAD

**Issue by ehuss** [raphlinus/pulldown-cmark#393]
The following links don't work within blockquotes, though I would expect them to work since they work outside of blockquotes (and they work in other implementations like dingus and github).  The spec seems to imply that the leading `>` should be deleted and the paragraphs treated like they normally would.

```markdown
> [foo
> bar]

[foo bar]: http://example.com

> [foo][bar
> baz]

[bar baz]: http://example.com
```

Inline links seem to work OK:

```markdown
> [foo
> bar](http://example.com)
```

Tested latest master (3c9cdda33db9d7cd3fb074b533d415fac42d4905).

cc #301 which is kinda similar.

**Comment by marcusklaas** [comment]
I can't test this myself right now, but I believe this PR should take care of this: https://github.com/raphlinus/pulldown-cmark/pull/386

**Comment by ehuss** [comment]
Oh, indeed it does!  Sorry, I thought I searched through all the issues, but that one didn't come up.  Seeing as this is almost identical, and fixed by #386, I'll close this. Thanks so much!
